align/pnr/checkers.py

```diff
@@ -114,7 +114,7 @@ def f( gen, value, tag=None):
         blk = cblk.instance[cblk.selectedInstance]
         found = False
         if json_dir is not None:
-            pth = pathlib.Path( json_dir + "/" + blk.master + ".json")
+            pth = pathlib.Path( json_dir + "/" + blk.lefmaster + ".json")
```

Comment:
@stevenmburns After the commit, these is no error locally and layout looks correct.

Reply:
I used schematic2layout.py and there is no error